crates/miden-testing/tests/integration/scripts/mod.rs

```diff
@@ -3,3 +3,5 @@ mod p2id;
 mod p2idr;
 mod send_note;
 mod swap;
+
+mod in_flight_swap;
```

Comment:
```suggestion
mod swap;
mod in_flight_swap;
```

Nit: Remove newline for consistency.